src/user/views/EditProfile/EditProfile.tsx

```diff
@@ -17,10 +17,7 @@ export const EditProfileView: React.FC = () => {
       <FloatContentWrapper>
         <EditProfileCard />
         <div style={{ textAlign: "center", marginTop: "1.5rem" }}>
-          <GradientButton
-            style={{ width: "8.4375rem", color: "#fff", fontWeight: "bold" }}
-            onClick={handleSaveProfileChanges}
-          >
+          <GradientButton style={{ width: "8.4375rem" }} onClick={handleSaveProfileChanges}>
```

Comment:
avoid inline styles

Reply:
That's because the button has a default width size nad the desing has another one, I'm trying to re-use the existing one without create a new one